alica_tests/src/test/etc/plans/ConfigurationTestPlan.pml

```diff
@@ -40,52 +40,7 @@
   "states": [
     {
       "comment": "",
-      "confAbstractPlanWrappers": [
```

Comment:
can delete this file right?

Reply:
File?

Reply:
yes, since Configurations are gone, we don't need any tests related to it